GoogleMapsApi.Test/packages.config

```diff
@@ -1,5 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Why did you delete it?

Reply:
The Visual Studio 2019 just included thus file into csproj file.